### PR TITLE
Add implementation of InciSe

### PR DESCRIPTION
**Pull request by dboun**
Separate application with only specific plugins enabled, but follows CaPTk on everything else.

Resolved<s>Note: PR #54 has to be merged into master alongside this for everything to work properly</s>

### CODE REVIEW

**Review by AlexanderGetka-cbica — APPROVED** [review]

File: Applications/AppList.cmake [hunk 1]
```diff
@@ -9,6 +9,7 @@
 # allow early dependency checking
 
 option(MITK_BUILD_APP_CaPTkApp "Build the MITK CaPTkApp" ON)
+option(MITK_BUILD_APP_CaPTkInciSe "Build the CaPTkInciSe application" ON)
```
Comment (ashishsingh18) [inline Applications/AppList.cmake]: Isn't the name "InciSe" only?

File: Applications/CaPTkInciSe/MitkCaPTkInciSe.cpp [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+#ifdef _MSC_VER
+#    pragma comment(linker, "/subsystem:windows /ENTRY:mainCRTStartup")
+#endif
+
+int main(int argc, char **argv)
+{
+  mitk::BaseApplication app(argc, argv);
+
+  app.setApplicationName("CaPTkInciSe");
```
Comment (ashishsingh18) [inline Applications/CaPTkInciSe/MitkCaPTkInciSe.cpp]: Isn't the name "InciSe"?

File: Applications/AppList.cmake (hunk 1 above)
Comment (dboun) [reply Applications/AppList.cmake]: done

**Comment by dboun** [comment]
@ashishsingh18 renaming done. sorry that it is in 2 commits instead of one so you can't diff easily

**Review by ashishsingh18 — APPROVED** [review]
looks okay.

**Pull request merged**